src/main/java/dev/chatcodes/ChatCommentModel.java

```diff
@@ -64,6 +64,7 @@ public void clearAllComments() {
             CommentLocation location = entry.getKey();
             removeComment(location);
         }
+        lineComments.clear();
```

Comment:
LOL, we started off trying to test this. Got distracted by the bit above. And end up making the colours work :)

Reply:
oh yeah, we have no idea if this worked :D